docs/sources/tempo/metrics-generator/service_graphs/metrics-queries.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Service graph metrics queries
+menuTitle: Metrics queries
+description: Use PromQL queries to access metrics from service graphs
+weight: 300
+---
+
+# Service graph metrics queries
+
+A collection of useful PromQL queries for service graphs.
+
+In most cases, users want to see a visual representation of their service graph. Grafana uses the service graph metrics created by Tempo and builds that visual for the user. However, in some cases, users may want to interact with the metrics that define that service graph directly. They may want to, for example, programmatically analyze how their services are interconnected and build downstream applications that use this information. 
```

Comment:
I did a bit of a rewrite here. Let me know what you think Kim.

Reply:
Much improved. Thank you.